From 3a142711c4c5ec46988af243a8cd31cf1440e1f7 Mon Sep 17 00:00:00 2001
From: Magnus Kessler <Magnus.Kessler@gmx.net>
Date: Thu, 3 Aug 2023 14:23:40 +0200
Subject: [PATCH] Fix version number in setup.py

The setup.py version number should be the same as the package version
in the js/package*.json files. This ensures that external build systems
like OpenWRT can be configured with the correct release version.

Signed-off-by: Magnus Kessler <Magnus.Kessler@gmx.net>
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index d674cfe..9475037 100644
--- a/setup.py
+++ b/setup.py
@@ -32,7 +32,7 @@ class RemoteAccessBuild(build_py):
 
 setuptools.setup(
     name=NAME,
-    version='1.2.1',
+    version='1.3.0',
     packages=setuptools.find_packages(exclude=['tests']),
     include_package_data=True,
 
-- 
2.34.1

